graphstorm-processing/docker/0.2.2/sagemaker/Dockerfile.cpu

```diff
@@ -60,10 +45,11 @@ fi
 ENTRYPOINT ["bash", "/usr/lib/spark/code/docker-entry.sh"]
 
 FROM base AS prod
-RUN python3 -m pip install /usr/lib/spark/code/graphstorm_processing-*.whl && \
-    rm /usr/lib/spark/code/graphstorm_processing-*.whl
+RUN python3 -m pip install --no-deps /usr/lib/spark/code/graphstorm_processing-*.whl && \
+    rm /usr/lib/spark/code/graphstorm_processing-*.whl && rm -rf /root/.cache
 CMD ["gs-processing"]
 
 FROM base AS test
-RUN python3 -m pip install /usr/lib/spark/code/graphstorm-processing/
-CMD ["sh", "-c", "pytest ./code/tests/"]
+RUN python3 -m pip install --no-deps /usr/lib/spark/code/graphstorm-processing/ mock && \
```

Comment:
Same here, do we need to install pytest?

Reply:
Pytest is already installed on the image, but `mock` is missing.